README.md

```diff
@@ -180,10 +210,10 @@ WantedBy=multi-user.target⏎
 ### Upgrading to a new version
 To upgrade to a new version, go to the project root and run `git pull`.
 If something has changed on the models, run `source ./venv/bin/activate` to
```

Comment:
Maybe clarify you are talking about the database models: 
"If something has changed in the database models..."